### PR TITLE
Fix invite teammate

### PR DESCRIPTION
**Pull request by pablo014**

### CODE REVIEW

File: src/components/modals/InviteUsers.vue [hunk 1]
```diff
@@ -124,9 +124,10 @@ export default {
       );
       organizationResults.value = results.entities.organizations;
     }
-    async function inviteUsers() {
+    const inviteUsers = async () => {
       let tags = _.defaultTo([...usersToInvite.value], []);
       try {
+        console.log(tags, emails.value);
```
Comment (tabiodun) [inline src/components/modals/InviteUsers.vue:130]: Can you please remove the console log

Comment (pablo014) [reply src/components/modals/InviteUsers.vue]: done, totally forgot about that one

File: src/components/modals/InviteUsers.vue [hunk 2]
```diff
@@ -139,9 +139,9 @@ export default {
           await $toasted.error(t('inviteTeammates.provide_valid_email'));
           return;
         }
-        const emails = tags.map((value) => value.text);
+        const emailsGroup = tags.map((value) => value.text);
```
Comment (deepanchal) [inline src/components/modals/InviteUsers.vue:142]: Does renaming this variable fix the issue because `emails` is already defined in the global scope?

Comment (pablo014) [reply src/components/modals/InviteUsers.vue:142]: yes

**Pull request merged**